frontend/src/components/visual-editor/BlockDialog.tsx

```diff
@@ -69,24 +69,25 @@ const BlockDialog: FC<BlockDialogProps> = ({
       toast.success(t("message.success_save"));
     },
   });
-  const methods = useForm<IBlockAttributes>({
-    defaultValues: {
-      name: block?.name || "",
-      patterns: block?.patterns || [],
-      trigger_labels: block?.trigger_labels || [],
-      trigger_channels: block?.trigger_channels || [],
-      options: block?.options || {
-        typing: 0,
-        content: {
-          display: OutgoingMessageFormat.list,
-          top_element_style: "compact",
-          limit: 2,
-        },
-        assignTo: block?.options?.assignTo,
+  const DEFAULT_VALUES = {
+    name: block?.name || "",
+    patterns: block?.patterns || [],
+    trigger_labels: block?.trigger_labels || [],
+    trigger_channels: block?.trigger_channels || [],
+    options: block?.options || {
+      typing: 0,
+      content: {
+        display: OutgoingMessageFormat.list,
+        top_element_style: "compact",
+        limit: 2,
       },
-      assign_labels: block?.assign_labels || [],
-      message: block?.message || [""],
+      assignTo: block?.options?.assignTo,
     },
```

Comment:
Can we add some defaults for context var capture and local fallback ?

Reply:
Sure